bHY6yh-de3608/cod-pu6y7d_bHY6yh-de3608.xml

```diff
@@ -82,67 +82,66 @@
             </p>
             <p xml:id="bHY6yh-de3608-d1e164">
                <lb ed="#V" n="28"/>
-               <quote xml:id="bHY6yh-de3608-Qd1e168">¶ Paralipomenon li. i. ca. 28. Si quaesieris dominum
+               ¶ <ref>Paralipomenon li. i. ca. 28.</ref> <quote xml:id="bHY6yh-de3608-Qd1e168">Si quaesieris dominum
         <lb ed="#V" n="29"/>inuenies: si autem dereliquaris eum proijciet te in
         <lb ed="#V" n="30"/>eternum. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e176"> /
-      <quote xml:id="bHY6yh-de3608-Qd1e178"> Libro. 2 c. 15. Si quasieriti deum inue
+       <ref>Libro. 2 c. 15.</ref> <quote xml:id="bHY6yh-de3608-Qd1e178">Si quasieriti deum inue
         <lb ed="#V" n="31"/>nietis: si autem dereliquiritis eum: derelinquit vos. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e184">
                <lb ed="#V" n="32"/>
-               <quote xml:id="bHY6yh-de3608-Qd1e188">¶ hieremias propheta in li. prophetarum. ca. 29
-        <lb ed="#V" n="33"/>Cum quefieritis me in toto corde vestro in
+               ¶ <ref>hieremias propheta in li. prophetarum. ca. 29.</ref>
+        <lb ed="#V" n="33"/> <quote xml:id="bHY6yh-de3608-Qd1e188">Cum quefieritis me in toto corde vestro in
         <lb ed="#V" n="34"/>ueniar a vobis: ait dominus. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e197">
                <lb ed="#V" n="35"/>
-               <quote xml:id="bHY6yh-de3608-Qd1e201">¶ Iesus filius sirach in eccl. c. 2. Deuns prote
+               ¶ <ref>Iesus filius sirach in eccl. c. 2.</ref> <quote xml:id="bHY6yh-de3608-Qd1e201">Deus prote
         <lb ed="#V" n="36" break="no"/>ctor est omnibus exquirentibus se in veritate. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e207">
                <lb ed="#V" n="37"/>
-               <quote xml:id="bHY6yh-de3608-Qd1e211">¶/ Pylon. li. sapiente. c. i. Sentite de domino: in
+               ¶/ <ref>Pylon. li. sapiente. c. i.</ref> <quote xml:id="bHY6yh-de3608-Qd1e211">Sentite de domino: in
         <lb ed="#V" n="38"/>bonitate: et in simplicitate cordis quarite illum
         <lb ed="#V" n="39"/>quoniam inuenitur ab his qui non tentant illum: appa
         <lb ed="#V" n="40"/>ret autem illis qui fidem habent in illum. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e221">
                <lb ed="#V" n="41"/>
-               <quote xml:id="bHY6yh-de3608-Qd1e225">¶ Matt hc. in euangel. c. 7. Querite et inueni
+               ¶ <ref>Matt hc. in euangel. c. 7.</ref> <quote xml:id="bHY6yh-de3608-Qd1e225">Querite et inueni
         <lb ed="#V" n="42"/>etis: pulsate et aperietur vobis. </quote>
             </p>
             <p xml:id="bHY6yh-de3608-d1e231"> /
-      <quote xml:id="bHY6yh-de3608-Qd1e233"> Idem di
+      <quote xml:id="bHY6yh-de3608-Qd1e233"> Idem di <!--confused about this it a ref without a quote?-->
```

Comment:
We'll need to double check but I think this is an instance where Matthew and Luke have the same "text". So he is saying "idem" (the same quote as above), but in Luke. So in this case there is no quote, but only a ref.